### PR TITLE
MEN-6086: docs: basic readme content

### PR DESCRIPTION
**Pull request by 0lmi**
Changelog: None
Ticket: None
Signed-off-by: Alex Miliukov <oleksandr.miliukov@northern.tech>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Mender Application Updates
+
+The repo contains Mender Update Modules for supporting containerized application updates on devices.
```
Comment (merlin-northern) [inline README.md:3]:
```suggestion
The repo contains Mender Update Module and artifact generator
```

**Review by merlin-northern — COMMENTED** [review]
looks good, just one addition

File: README.md (hunk 1 above)
Comment (0lmi) [reply README.md:3]: @merlin-northern what are the plans for future? Will the repo contain only those or it's a plans for something more?

Comment (merlin-northern) [reply README.md:3]: at one point we will provide support for applications based on something else than containers. for now we focus on container-based deployments.

**Review by merlin-northern — APPROVED** [review]

**Pull request merged**